testdata/bad/backslash_at_end_of_line_in_DataTable.feature

```diff
@@ -0,0 +1,5 @@
+Feature: Addition
+Scenario: Add two numbers
+	When I press
+		| foo |
+		| bar \
```

Comment:
Think this file should be snake cased and there is a newline missing here

Reply:
I have changed all the testdata file names.

Regarding the missing newline: I think this is part of the tests because they should resemble someone editing the feature files in an incomplete state. The gherkin parser should parse them. So I think this is OK in this case?

I copied the tests from the [reqnroll ide plugin](https://github.com/reqnroll/Reqnroll.VisualStudio/blob/main/Tests/Reqnroll.VisualStudio.Tests/Editor/Services/DeveroomGherkinParserTests.cs).